src/chart-addons/stacked-legend.js

```diff
@@ -306,7 +314,69 @@ const formatNumber = d => (String(d).length > 4 ? format(".2s")(d) : commafy(d))
 const color_literal_alpha_regex = /^\s*[a-z,A-Z]{3}[aA]\s*\([\d.]+,[\d.]+,[\d.]+,([\d.]+)\)$/i
 
 // eslint-disable-next-line complexity
-function legendState(state, useMap = true) {
+function legendState_v1(state, useMap) {
```

Comment:
this completely restored the original logic and is used for version 1-tagged input state